draft-ietf-oauth-device-flow.xml

```diff
@@ -646,14 +654,17 @@
           inform the user that they are authorizing a device during the user
           interaction step (see <xref target="UserInteraction"/>), and to
           confirm that the device is in their possession.
+	  The authorization server SHOULD display information about the device
+	  so that the person would notice if a software client
```

Comment:
would -> could?

The last IESG review I went through, my doc was criticized for putting too much faith in the user. I think with this SHOULD the user has the ability to notice, but may miss the signal.